localization/english/events/event_localization/spark_main_story_events_l_english.yml

```diff
@@ -1,43 +1,43 @@
+  elf_destiny_main_story_events.001.a:0 "An adventure? You have my support!"
+  elf_destiny_main_story_events.001.b:0 "The Elves? Don't waste my time. Nothing but a children's tale!"
 
   elf_destiny_main_story_events.002.t:0 "Restore the Grand Portal"
-  elf_destiny_main_story_events.002.desc:0 "Relics of old collected and my worthiness unquestionable. \n\nI now posses the means to re-establish the bond between worlds and establish my self as a true Realm lord.\n\n There are many who will condemn me and would move to halt this great work by any means.\n\nAm I ready for this undertaking?"
+  elf_destiny_main_story_events.002.desc:0 "Relics of old collected and my worthiness unquestionable. \n\nI now posses the means to re-establish the bond between worlds and establish myself as a true Realm lord.\n\nThere are many who will condemn me and would move to halt this great work by any means.\n\nAm I ready for this undertaking?"
   elf_destiny_main_story_events.002.a:0 "We break ground at once!"
-  elf_destiny_main_story_events.002.b:0 "More preparations are required..."
+  elf_destiny_main_story_events.002.b:0 "More preparations is required..."
 
   elf_destiny_main_story_events.0021.t:0 "Thanks For Playing!"
   elf_destiny_main_story_events.0021.desc:0 "Sorry, thats all the story content so far! More is under construction and on the way. Visit the official Discord to keep up with updates! \n\n I still want to reward you for playing this far, so pick what you would like! Thanks Again!"
 
   elf_destiny_main_story_events.0021.a:0 "A Free Blessing"
   elf_destiny_main_story_events.0021.b:0 "Gold And Prestige"
-  elf_destiny_main_story_events.0021.c:0 "Im odd and want for nothing"
+  elf_destiny_main_story_events.0021.c:0 "I'm odd and want nothing."
 
-  elf_destiny_main_story_events.003.desc:0 "[scoped_ruler.GetTitledFirstName] has restored an some kind of ancient portal in the far north.\n\nLegends say this could allow [scoped_ruler.GetHerHim] to communicate with the stars and bring a people old as time to this world.\n\nThe known paradigm of the world could be about to change."
+  elf_destiny_main_story_events.003.desc:0 "[scoped_ruler.GetTitledFirstName] has restored some kind of an ancient portal in the far north.\n\nLegends say this could allow [scoped_ruler.GetHerHim] to communicate with the stars and bring a people old as time into this world.\n\nThe existing paradigm of our world could be about to change."
```

Comment:
I dont know if this first "an" is needed, I know had it but maybe we can just drop it all together

Reply:
Works with or without, but I personally think it flows better with 'an'. I suppose since it's a unique structure just 'ancient portal' would do, but the event is written from the point of someone who doesn't know what exactly it is so they wouldn't really know it's THE ancient portal.